internal/storage/gcs/errors.go

```diff
@@ -69,6 +72,11 @@ func GetGCSError(err error) error {
 			return &NotFoundError{Err: err}
 		case codes.FailedPrecondition:
 			return &PreconditionError{Err: err}
+		case codes.Aborted:
+			errMsg := rpcErr.Message()
+			if strings.Contains(errMsg, takeoverErrMsg) {
```

Comment:
not sure if we should match the entire string. any small word change will break us.
@abhishek10004  - thoughts?